src/basic/cgroup-util.h

```diff
@@ -115,12 +115,24 @@ extern const uint64_t cgroup_io_limit_defaults[_CGROUP_IO_LIMIT_TYPE_MAX];
 const char* cgroup_io_limit_type_to_string(CGroupIOLimitType t) _const_;
 CGroupIOLimitType cgroup_io_limit_type_from_string(const char *s) _pure_;
 
+/* Special values for the cpu.idle attribute */
+#define CGROUP_CPU_IDLE_INVALID INT64_MIN
+#define CGROUP_CPU_IDLE_DEFAULT INT64_C(0)
+#define CGROUP_CPU_IDLE_MIN 0
+#define CGROUP_CPU_IDLE_MAX 1
```

Comment:
it's not really a number. the kernel interface is just weird, but let's not expose it like that. make it a boolean, a tri-state (i.e. a boolean encoded in an `int`, which has the additional value -1 for "unset") or a proper enum. But definitely not an int64_t.